types/react-editable-label/index.d.ts

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+
+export interface EditableLabelProps {
+    initialValue: string;
+    save: (value: string) => void;
+    disableKeys?: boolean;
+    inputClass?: string;
+    labelClass?: string;
+    inputName?: string;
+    inputId?: string;
+}
+
+export function EditableLabel(props: EditableLabelProps): React.JSX.Element;
```

Comment:
Is this actually a named export? This looks like it'll be a default export (or `export =`?): https://github.com/markbiek/react-editable-label/blob/master/lib/EditableLabel.js#L93

But, not sure what the bundle looks like.

Reply:
Can you give me a little more detail? I'm not sure I completely understand what you're saying 😄.

Reply:
Your readme says:

```
import EditableLabel from 'react-editable-label';
```

But this package's tests do:

```
import { EditableLabel } from 'react-editable-label';
```

Which are very different things. The declaration needs to have a `export default EditableLabel`, or more likely, an `export = EditableLabel`.

Reply:
Ah, I see what you're referring to. The only way I could get the DefinitelyTyped CI to pass was to change that export in the `index.d.ts`. When I move things to my package, I make sure the exports match.

Reply:
Right, but we can't merge it here if it was still wrong, either.